bundle/config/resources_test.go

```diff
@@ -168,6 +170,11 @@ func TestResourcesBindSupport(t *testing.T) {
 				CreateServingEndpoint: serving.CreateServingEndpoint{},
 			},
 		},
+		SecretScopes: map[string]*resources.SecretScope{
+			"my_secret_scope": {
+				Name: "0",
```

Comment:
Is there significance to "0"?

Reply:
"0" is an example string id that is safely parsed by `strconv.Atoi` (jobs call this as part of their `Exists` implementation). It can be replaced with any integer for this test purpose

Reply:
Why does it need to be parsed by `strconv.Atoi`?

Reply:
Ack, thanks.